rudder-agent/SOURCES/patches/fusioninventory/11726-Extend-Rudder-specific-inventory-with-clie.patch

```diff
@@ -122,6 +122,11 @@ index b2fd04d..41e1dea 100644
 +                };
 +                if ($@) {
 +                    $logger->error("Script $script_file didn't return valid JSON entry, error is:$@") if $logger;
++                    my $filename = "/tmp/inventory-json-error-".time();
```

Comment:
maybe `/var/rudder/tmp/`, or at least add rudder in the name and make it only readable by root, we don't know what these variables contains.

Reply:
@peck ping